contentcuration/contentcuration/frontend/channelEdit/views/ResourcePanel.vue

```diff
@@ -472,23 +472,28 @@
       /* VALIDATION */
       // License isn't specified
       noLicense() {
-        return !this.isTopic && validateNodeLicense(this.node).length;
+        return Boolean(!this.isTopic && validateNodeLicense(this.node).length);
       },
       // Copyright holder isn't set on non-public domain licenses
       noCopyrightHolder() {
-        return !this.isTopic && !validateNodeCopyrightHolder(this.node).length;
+        return Boolean(!this.isTopic && validateNodeCopyrightHolder(this.node).length);
       },
       // License description isn't provided on special permissions licenses
       noLicenseDescription() {
-        return !this.isTopic && !validateNodeLicenseDescription(this.node).length;
+        return Boolean(
+          this.license &&
+            this.license.is_custom &&
+            !this.isTopic &&
+            validateNodeLicenseDescription(this.node).length
+        );
       },
       // Invalid mastery model
       noMasteryModel() {
         return (
           this.isExercise &&
-          (!validateNodeMasteryModel(this.node).length ||
```

Comment:
Nice, thanks for catching this

Reply:
I'm sorry for the first comment. This change needs to be reverted. Mastery model error is not showing up in the resource panel because of it.

A template for an error is:
```
<span v-if="noMasteryModel" class="red--text">
   <Icon color="red" small>error</Icon>
   <span class="mx-1">{{ $tr('noMasteryModelError') }}</span>
</span>
```

It should be displayed when mastery model is not valid, which happens when at least one of `validateNodeMasteryModel`,  `validateNodeMasteryModelM` or  `validateNodeMasteryModelN` returns an array with at least one error in it ([all `validate...` functions](https://github.com/learningequality/studio/blob/develop/contentcuration/contentcuration/frontend/shared/utils/validation.js#L139) return arrays of error codes if any found)

Reply:
We should check for errors not the absence of them, right? If there is no length, then there is no error so if any of those arrays have a length then we should return `true` in that sequence of `ors` and that's how the PR is set to work.

Definitely would like to see the functions renamed though for sure to help avoid the confusion.